sample/src/main/java/com/canhub/cropper/sample/camera_java/app/SCameraFragmentJava.java

```diff
@@ -0,0 +1,338 @@
+package com.canhub.cropper.sample.camera_java.app;
+
+import android.Manifest;
+import android.app.Activity;
+import android.app.AlertDialog;
+import android.content.ActivityNotFoundException;
+import android.content.Context;
+import android.content.DialogInterface;
+import android.content.Intent;
+import android.graphics.Bitmap;
+import android.graphics.Color;
+import android.net.Uri;
+import android.os.Bundle;
+import android.os.Environment;
+import android.provider.MediaStore;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import androidx.activity.result.ActivityResultLauncher;
+import androidx.activity.result.contract.ActivityResultContracts;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.core.content.FileProvider;
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentActivity;
+import androidx.viewpager.widget.ViewPager;
+
+import com.canhub.cropper.CropImage;
+import com.canhub.cropper.CropImageView;
+import com.canhub.cropper.sample.SCropResultActivity;
+import com.canhub.cropper.sample.camera.domain.CameraEnumDomain;
+import com.canhub.cropper.sample.camera.domain.SCameraContract;
+import com.canhub.cropper.sample.camera.presenter.SCameraPresenter;
```

Comment:
Should not implement `camera` package, please create in java for `camera_java` package.

If we change one package later this should not impact another one.